src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
```

Comment:
You could use `space-between` property here and add smaller max-width. Then you can avoid using `margin-left` on card. And so that there will always be same distance between cards